tests/system/action/motion/test_create_sequential_number.py

```diff
@@ -144,8 +146,13 @@ def test_create_sequential_numbers_deleted_motion(self) -> None:
         model = self.get_model("motion/2")
         self.assertEqual(model.get("sequential_number"), 2)
 
+    @pytest.mark.skip(
+        reason="Not working with watch-thread, because the testlock is unknown in action_worker"
+    )
     def test_create_sequential_numbers_race_condition(self) -> None:
         """
+        !!!We could delete this test or implement a switch-off for the action_worker procedure at all!!!
```

Comment:
I would support the second option. Maybe with a header like `disable-action-worker`? I think this test is valuable and should be kept.

Reply:
This is not as easy as it seems to be in the initial part of the job, because each action is processed in the action_worker-thread.
There is a new issue issue #1473 to fix this, because for now we want the functionality of the thread watcher